docs/index.md

```diff
@@ -22,9 +22,7 @@ see [Getting started with the BigAnimal free trial.](https://www.enterprisedb.co
 ```terraform
 # Configure the BigAnimal Provider
 provider "biganimal" {
-  ba_bearer_token = "<redacted>"
```

Comment:
We need to modify the example file too https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/main/examples/provider/provider.tf. Remove bearer token in the example file and also remove the access key comments

Reply:
@wai-wong-edb : Done. Please check.